app/build.gradle.kts

```diff
@@ -198,9 +197,6 @@ dependencies {
     val NAV_VERSION = "2.7.1"
     val BILLING_VERSION = "6.0.1"
 
-    // TODO: re-evaluate when RxJava is completely replaced with kotlin co-routines
```

Comment:
I wasn't sure what to check regarding this TODO. Let me know if I missed something